packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -25,6 +25,8 @@ export interface TranslateWordProps {
   referenceGloss?: string;
   suggestions: string[];
   onChange(data: { gloss?: string; approved?: boolean }): void;
+  onShowInSidebar?: () => void;
+  onOpenSidebar?: () => void;
```

Comment:
This component shouldn't need to know that we have a sidebar. Can we call this `onFocus` and `onShowDetail`?